src/airodump-ng/airodump-ng.c

```diff
@@ -6530,7 +6577,22 @@ int main(int argc, char * argv[])
 				}
 				else
 				{
-					printf("Notice: invalid bssid\n");
+					printf("Error: invalid bssid\n");
```

Comment:
BSSID should be uppercase

Reply:
fixed

Reply:
updated it for --bssid also, so it is consistent